spec/features/tests/show_view_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+feature "user views show page", %{
+  As a user
+  I want to view a food truck's information
+  so that I can learn more about it
+} do
+  # Acceptance Criteria
+  # * If a user goes to the show page, they see a list of food trucks.
+
+  scenario "user views show page of a food truck" do
+    sample_food_truck_1 = FoodTruck.create(
+      name: "Chicken & Rice Guys",
+      description: "Food truck that specializes in chicken and lamb shawarma.",
+      avg_rating: 4,
+      location: "Harvard Square"
+    )
+
+    visit food_truck_path(sample_food_truck_1.id)
+    expect(page).to have_content(sample_food_truck_1.name)
+    expect(page).to have_content(sample_food_truck_1.description)
+    expect(page).to have_content(sample_food_truck_1.avg_rating)
+    expect(page).to have_content(sample_food_truck_1.location)
+
+  end
+end
```

Comment:
It looks to me like you built in a link to a given show page from the index. A test that this link works would be appropriate here, in addition to the one above that navigates to the path directly.